src/main/java/plugily/projects/buildbattle/arena/states/build/InGameState.java

```diff
@@ -129,11 +131,18 @@ public void handleCall(PluginArena arena) {
 
   private void givePlaceRewards(BuildArena pluginArena) {
     RewardsFactory rewards = getPlugin().getRewardsHandler();
+    RewardType placeRewardType = null;
```

Comment:
already do rewards.getRewardType("PLACE") there?

Reply:
Useless to pre-cache the reward type if for example the `list` below is null or empty.

Reply:
its unlikly that the list would be null as if its null there won't be players right?